src/Raven.Server/Config/Categories/SecurityConfiguration.cs

```diff
@@ -203,6 +199,11 @@ internal static void Validate(RavenConfiguration configuration)
                     if (isServerUrlHttps)
                         throw new InvalidOperationException($"Configured server address { string.Join(", ", configuration.Core.ServerUrls) } requires HTTPS. Please set up certification information under { RavenConfiguration.GetKey(x => x.Security.CertificatePath) } configuration key.");
 
+                    var serverAddresses = DetermineServerIp(uri);
```

Comment:
This can also throw, no?

Reply:
Yes. But I don't think we cannot just ignore this. We want to use the returned IP to verify it using `SecurityUtils.IsUnsecuredAccessAllowedForAddress` here:

https://github.com/ravendb/ravendb/blob/5213dbcc9cbe2b6e24960ce1fac4281bbc69211e/src/Raven.Server/Config/Categories/SecurityConfiguration.cs#L194